tests/integration/helpers/helpers.go

```diff
@@ -0,0 +1,203 @@
+package helpers
```

Comment:
This file is copied directly from the `integration-testing` private repo. Only the `ExecInContainer` function is created by me.

Reply:
I see everyone has to copy this file for its own integration tests. Would not be better having a kind of `integration tests` folder in the sdk to avoid duplicating this code in the different repositories?

Reply:
I Agree. Do you think the `infra-integrations-sdk` is the best place for putting that, or maybe another repository?

Reply:
Good question. Would you expect as a SDK user having this tools for creating your own integrations tests? If answer is no then another repository, but it's yes then:

PRO:
- having these tools in SDK would help users create integration tests for new integrations which I think is a good idea.

CONS:
- More files in SDK which ends up with more complexity for the project.

If we locate this files in a very clear path would make sense FMPOV, but we could talk with people who created the SDK who have better background than me.

Reply:
Maybe we can start with a common `ohi-testing-tools` repository, and always use master branch for developing the current MMF without blocks (making sure we don't break APIs).

And eventually decide to move all the stuff there to a more stable version in the SDK repo...

Reply:
Let's define this with product and see when we can do this.